mhlo_builder/csrc/mlir/mhlo/builder/element_wise_binary.h

```diff
@@ -28,7 +28,26 @@ static constexpr const char kCompare_GT[] = "GT";
 static constexpr const char kCompare_LE[] = "LE";
 static constexpr const char kCompare_LT[] = "LT";
 
-mhlo::ComparisonDirection inline getComparisonDirectionFromString(
+chlo::ComparisonDirection inline getChloComparisonDirectionFromString(
+    std::string cmp) {
+  if (cmp == kCompare_EQ) {
+    return chlo::ComparisonDirection::EQ;
+  } else if (cmp == kCompare_NE) {
+    return chlo::ComparisonDirection::NE;
+  } else if (cmp == kCompare_GE) {
+    return chlo::ComparisonDirection::GE;
+  } else if (cmp == kCompare_GT) {
+    return chlo::ComparisonDirection::GT;
+  } else if (cmp == kCompare_LE) {
+    return chlo::ComparisonDirection::LE;
+  } else if (cmp == kCompare_LT) {
+    return chlo::ComparisonDirection::LT;
+  } else {
+    assert(false && "Unhandled comparison direction.");
```

Comment:
better to have a `CHECK`, `assert` will be eliminated by compiler and thus undefined value in this branch.

Reply:
done